backend/controllers/roomControllers.js

```diff
@@ -11,4 +12,22 @@ async function findingRoom(req, res) {
   }
 }
 
+async function getCategories(req, res) {
+  try {
+    const categories = await categoryModel.find();
+
+    const formattedCategories = categories.map((category) => ({
+      id: category._id,
+      name: category.categoryName,
+    }));
+
+    res.status(200).json({
+      message: 'successfully',
+      categories: formattedCategories,
+    });
+  } catch (err) {
+    res.status(500).json({ message: 'error from server' });
```

Comment:
Để lại cái is_err cho thêm redundancy cũng đc mà. Giống #16 ấy
```suggestion
    res.status(500).json({ is_err: true, message: 'Internal Server Error' });
```

Reply:
Hoặc là sửa bên #16 cho tiện đi
```suggestion
    res.status(500).json({ message: 'Internal Server Error' });
```